src/BizHawk.Tests/Client.Common/lua/LuaTests.cs

```diff
@@ -0,0 +1,874 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+using BizHawk.Client.Common;
+using BizHawk.Common.StringExtensions;
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace BizHawk.Tests.Client.Common.Lua
+{
+	[TestClass]
+	public class LuaTests
+	{
+		private static readonly NLua.Lua LuaInstance = new();
+		private static readonly NLuaTableHelper _th = new(LuaInstance, Console.WriteLine);

```

Comment:
Can you subclass `LuaLibraryBase` to get this stuff?

Reply:
Doesn't work. LuaLibraryBase doesn't have an NLua.Lua in the first place, and the NLuaTableHelper is just obtained using the IPlatformLuaLibEnv passed in the ctor.